src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/remove_convert.cpp

```diff
@@ -111,6 +115,19 @@ TEST_P(RemoveUselessConvertCPUTest, CompareWithRefs) {
     CheckNumberOfNodesWithType(compiledModel, "Convert", 0);
 }
 
+using RemoveUselessFP16ConvertCPUTest = RemoveUselessBF16ConvertCPUTest;
+TEST_P(RemoveUselessFP16ConvertCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    if (!(ov::with_cpu_x86_avx512_core_fp16() || ov::with_cpu_x86_avx512_core_amx_fp16())) {
+        GTEST_SKIP() << "Skipping test, platform don't support precision f16";
+    }
```

Comment:
Do we really have ISA dependency for this test?

Reply:
already updated it